scripts/helpers/restyle-diff.sh

```diff
@@ -45,33 +46,67 @@ docker_run() {
 }
 
 restyle-paths() {
-
     image=restyled/restyler:edge
+    batch_size=4
```

Comment:
How about using `$(nproc)`? or is that too much?

Do these restylers support multiple paths in some way?

Reply:
I tried locally just passing in all files and it seemed to work. The worst may be if we somehow have too many files changed (regen?) and they do not fit in max command line arguments. We should either just pass all (first try) or pass in large batches that are configurable (like 256 or 1024 files at once or something)

Reply:
Is there a way to get command line arguments from a file? I would think so.

BTW on linux, max command line length is ~100KiB

Reply:
we can xargs it with a large count like 256 and just assume it is ok. My `getconf ARG_MAX` on my machine is about 2MB. Or even 100 files at a time will make most cases just be ok, even the codegen ones.

Reply:
ok i just added xargs, makes the script much more concise...

Reply:
i made it accept 256, tested it with some old branch and it isnt crashing so looks good